src/lib/jsonable/jsonable.ml

```diff
@@ -0,0 +1,60 @@
+open Core_kernel
+
+module type Converter_intf = sig
```

Comment:
I'm biased because I witnessed a big argument in the Swift universe on naming for this sort of thing and I like the following names:

```ocaml
module type Iso_intf = sig
   type from
   type to [@@deriving yojson]

  val encode : from -> to
  val decode : to -> from
end
```

Reply:
`to` is a reserved token in OCaml so I can't declare `to` as a type.

I am going to change the names `to` and `from` to `original` and `standardized`, respectively.